leapp/cli/report/__init__.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+from leapp.exceptions import CommandError
+from leapp.utils.audit import get_connection
+from leapp.utils.clicmd import command, command_opt
+from leapp.cli.upgrade import fetch_last_upgrade_context
+
+
+def fetch_upgrade_report_messages(context_id):
+    """
+    :return: All upgrade messages of type "LeappReport" withing the given context
+    """
+    with get_connection(None) as db:
+        cursor = db.execute(
+            "SELECT message_data_hash FROM message WHERE type = 'CheckResult' and context = '%s'" % context_id)
+        reports = cursor.fetchall()
+        report_msg_ids = [report[0] for report in reports]
+        cursor = db.execute(
+            "SELECT data FROM message_data WHERE hash in (%s)" % ','.join('?'*len(report_msg_ids)), report_msg_ids)
```

Comment:
Need to find better way here as sqlite will accept only up to 999 hash parameters I guess